doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc

```diff
@@ -58,4 +58,12 @@ kieservers:
 ----
 $ java -jar {PRODUCT_FILE}-process-migration-service-standalone.jar -s./config.yml
 ----
+. To enable Thorntail to auto detect a JDBC driver, add the `JAR` file with the driver name to the `thorntail.classpath` system property. For example, execute the following command:
```

Comment:
[style] If possible without losing context, can we avoid "to enable *subject*" as this anthropomorphizes Thorntail.
[style] Maybe split or simplify "For example, execute the following command:" to "For example:"?
[style] Maybe simplify "add the `JAR` file with the driver name"?

s/To enable Thorntail to auto detect a JDBC driver, add the `JAR` file with the driver name to the `thorntail.classpath` system property. For example, execute the following command:/To enable auto-detection of a JDBC driver by Thorntail, add the `JAR` file of the JDBC driver to the `thorntail.classpath` system property. For example:

Reply:
Done
Nice suggetions. Thank you.